publish-pre-release-cli.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+const _ = require('lodash');
+const fs = require('fs');
+const branchHelper = require('./src/helpers/branch-helper');
+
+const { runShellVerbose } = require('./run.js');
+
+const series = async (...commands) => {
+    for (const command of commands) {
+        await runShellVerbose(...command);
+    }
+};
+
+let newVersion = '';
+let tagName = '';
+
+const generateTagName = async () => {
+    const currentBranch = await branchHelper.getCurrentBranch(__dirname);
+    const branchName = branchHelper.getTicketNumberFromBranch(currentBranch);
+
+    const packageJSON = JSON.parse(fs.readFileSync(`${process.cwd()}/package.json`, 'utf8'));
+    const currentVersion = _.get(packageJSON, 'version');
+
+    if (currentVersion.includes(branchName) && currentVersion.match(/.*[0-9]+-[0-9]+$/) ){
+        newVersion = currentVersion.replace(/\d+$/, (i) => parseInt(i) + 1);
+    }
+    else{
+        newVersion = currentVersion + '-' + branchName + '-1';
+    }
+
+    packageJSON.version = newVersion;
+
+    fs.writeFile(`${process.cwd()}/package.json`, JSON.stringify(packageJSON, null, 2).concat('\n'), function(err) {
```

Comment:
You're using the package.json location both for reading and writing. It'd be nice if it was in a variable.